web3/eth.py

```diff
@@ -198,6 +200,25 @@ def getTransactionCount(self, account, block_identifier=None):
             ],
         )
 
+    def replaceTransaction(self, transaction_hash, new_transaction):
+        current_transaction = self.getTransaction(transaction_hash)
+        new_transaction = prepare_replacement_transaction(
+            self.web3, current_transaction, new_transaction
+        )
+
+        # TODO: Does the gas and from defaulting in `sendTransaction` make sense for this?
```

Comment:
Yes, in the replacement scenario, defaulting the `gas` and `from` fields normally seems reasonable.